cnd/src/storage.rs

```diff
@@ -1,18 +1,13 @@
 use crate::{
     asset,
     db::{
-        self,
-        tables::{Halight, Hbit, Herc20, SecretHash, Swap},
-        wrapper_types::custom_sql_types::Text,
-        ForSwap, NoHalightRedeemIdentity, NoHalightRefundIdentity, NoHerc20RedeemIdentity,
-        NoHerc20RefundIdentity, NoSecretHash, Save, Sqlite,
+        self, ForSwap, Halight, Hbit, Herc20, NoHalightRedeemIdentity, NoHalightRefundIdentity,
+        NoHerc20RedeemIdentity, NoHerc20RefundIdentity, NoSecretHash, Save, SecretHash, Sqlite,
+        Swap, Text,
     },
     halight, hbit, herc20, http_api, identity,
     network::{WhatAliceLearnedFromBob, WhatBobLearnedFromAlice},
-    seed::RootSeed,
-    spawn,
-    swap_protocols::state::Get,
-    LocalSwapId, Protocol, Role, Side, SwapContext,
+    spawn, Get, LocalSwapId, Protocol, Role, RootSeed, Side, SwapContext,
```

Comment:
:upside_down_face: **Very** minor nit, but importing `Get` by itself is not great. Maybe it doesn't matter because it's just in the imports section, but I would rather see something like `state::Get`.

Reply:
ok, I'm good with that. Will do.